apps/jobs/views.py

```diff
@@ -554,6 +554,7 @@ class ContactUsViewSet(viewsets.ModelViewSet):
 
     # queryset = ContactMessage.objects.all()
     serializer_class = ContactUsSerializer
+    http_method_names = ["get", "post"]
```

Comment:
To make sure the authorization check added `http_method_name` attribute which will by default deny other methods and only allow get and post methods.

Reply:
we dont need get method